src/uu/ls/src/ls.rs

```diff
@@ -1522,6 +1585,18 @@ fn display_item_long(
         }
     }
 
+    #[cfg(unix)]
+    {
+        if config.block_size.is_some() {
+            let block_size = config.block_size.map_or_else(get_env_block_size, |v| v);
+            let _ = write!(
```

Comment:
You are ignoring the result of `write!` here. This is usually not the best solution. On the other hand the surrounding code does the same, so I think it's fine (maybe there's a reason behind it, but I don't know)